drivers/adc/adc_stm32.c

```diff
@@ -589,6 +597,9 @@ static int start_read(const struct device *dev,
 	}
 
 	data->buffer = sequence->buffer;
+#ifdef CONFIG_ADC_STM32_DMA
+	data->buffer_size = sequence->buffer_size;
```

Comment:
data->buffer_size = buffer_size; ?

Reply:
Uh, there is no buffer_size parameter nor local var?

Reply:
OK, I missed the gap between changes. Nevermind then.